config_defaults_inc.php

```diff
@@ -4221,7 +4238,8 @@
 	'class_path','library_path', 'language_path', 'absolute_path_default_upload_folder',
 	'ldap_simulation_file_path', 'plugin_path', 'bottom_include_page', 'top_include_page',
 	'default_home_page', 'logout_redirect_page', 'manual_url', 'logo_url', 'wiki_engine_url',
-	'cdn_enabled', 'public_config_names', 'email_login_enabled', 'email_ensure_unique'
+	'cdn_enabled', 'public_config_names', 'email_login_enabled', 'email_ensure_unique',
+	'public_urls'
```

Comment:
It is unclear to me why it wouldn't be possible to set this configuration option via the database/UI.  We shouldn't limit this to just config_inc.php and hence global.